Update broadinstitute -> popgen links

illusional opened the pull request:
Addresses: https://github.com/populationgenomics/seqr/issues/10

Update:
- GH link broadinstitute/seqr -> populationgenomics/seqr in some places:
    - Left some references to preserve issue links
- Update some email addresses:
    - `seqr@broadinstitute...` to `seqr@populationgenomics...`
    - `matchmaker@broadinstitute...` to `seqr-matchmaker@popgen...`
    - Updated test emails like `test@broadinstitute...` (/ `test_user`) to `seqr+test1@popgen...`

TODO:

- [x] Update matchmaker@broad to matchmaker@popgen, even though matchmaker has been disabled for projects.

How do you update some links or text, like "matchbox":

![image](https://user-images.githubusercontent.com/22381693/114663133-30123e80-9d3d-11eb-87ab-e8d74b3af5c2.png)

lgruen commented on INCIDENTS.md:
```diff
@@ -1,10 +1,10 @@
-- 3/11/2020: seqr down after error on kubernetes redeployment
+Known, resolved incidents that occured on the CPG's seqr deployment (https://seqr.populationgenomics.org.au/): 
+- 
+W
```
```suggestion

```

lgruen commented on matchmaker/migrations/0001_initial.py:
```diff
@@ -165,7 +165,7 @@ class Migration(migrations.Migration):
                 ('label', models.CharField(blank=True, max_length=255, null=True)),
                 ('contact_name', models.TextField(default='Samantha Baxter')),
-                ('contact_href', models.TextField(default='mailto:matchmaker@broadinstitute.org')),
+                ('contact_href', models.TextField(default='mailto:seqr-matchmaker@populationgenomics.org.au')),
```
Actually not sure about this one! We should probably chat with Daniel about what the Matchmaker workflow is. At least currently we don't have a `seqr-matchmaker@populationgenomics.org.au` group, but it would be easy to create one.

lgruen reviewed (changes requested):
Can we check with Daniel about the Matchmaker process? I'm actually not sure whether this should be our group or stay with the Broad address.

illusional commented:
Hey @lgruen, I've updated the matchmaker emails as we discussed, and updated the test*@broad emails, but noticed some emails like: `UDNCC@hms.harvard.edu,matchmaker@phenomecentral.org` in some of the tests. Do we have a suggestion for this? I presume they could probably be removed (and tests fixed).

lgruen commented:
> Hey @lgruen, I've updated the matchmaker emails as we discussed, and updated the test*@Broad emails, but noticed some emails like: `UDNCC@hms.harvard.edu,matchmaker@phenomecentral.org` in some of the tests. Do we have a suggestion for this? I presume they could probably be removed (and tests fixed).

Is it difficult to fix those tests? In general I'm not a huge fan of tests that essentially check the value of a constant, compared to exercising functionality, so removing them is certainly an option.

illusional commented:
Closing in favour of #62
